pkgs/development/web/bun/default.nix

```
@@ -0,0 +1,19 @@
+{ stdenvNoCC, callPackage, lib }:
+let
+  version = "0.1.1";
+  pname = "bun";
+  meta = with lib; {
+    homepage = "https://bun.sh";
+    changelog = "https://github.com/Jarred-Sumner/bun/releases/tag/bun-${version}";
+    description = "Incredibly fast JavaScript runtime, bundler, transpiler and package manager – all in one.";
+    longDescription = ''
+      All in one fast & easy-to-use tool. Instead of 1,000 node_modules for development, you only need bun.
+    '';
+    license = licenses.mit;
+    maintainers = with maintainers; [ DAlperin ];
```

```suggestion
    maintainers = with maintainers; [ DAlperin jk ];
```

I'm happy to maintain this with you

---

Sure